.github/workflows/deploy-production.yml

```
@@ -79,7 +79,7 @@ jobs:
         with:
           github-token: ${{ secrets.WORKFLOW_CALL_TOKEN }}
       - uses: stampedeapp/actions/setup@main
-      - run: LDT_ENV=staging yarn predeploy || true
+      - run: LDT_ENV=staging if [ "$(cat package.json | jq -r .scripts.predeploy)" != "null" ]; then yarn predeploy; fi;
         env:
           MONGO_CONNECTION_STRING: ${{ secrets.MONGO_CONNECTION_STRING }}
```

```suggestion
```suggestion
      - id: predeploy
        name: Check for predeploy script
        run: echo "script=$(cat package.json | jq -r .scripts.predeploy)" >> $GITHUB_OUTPUT
      - run: ${{ steps.predeploy.outputs.script }}
        if: ${{ steps.predeploy.outputs.script }}
       name: Run predeploy script
        env:
          LDT_ENV: staging
          MONGO_CONNECTION_STRING: ${{ secrets.MONGO_CONNECTION_STRING }}
```